pkg/motor/motor.go

```diff
@@ -274,3 +276,83 @@ func (w *motorNodeImpl) SendTx(routeUrl string, msg sdk.Msg) ([]byte, error) {
 	}
 	return resp.GetTxResponse().Marshal()
 }
+
+func SetupTestAddressWithKeys(motor MotorNode) (error) {
```

Comment:
I don't think this is a good place for these test-related functions.

Reply:
agreed should be in `testutils`

Reply:
refactored